core/message-tags-3.3.md

```diff
@@ -176,3 +179,19 @@ In this example, plus signs, colons, equals signs and commas are transmitted raw
 ```
 @+example=raw+:=,escaped\:\s\\ :irc.example.com NOTICE #channel :Message
 ```
+
+---
+
+An empty message sent by a client with the `+example-client-tag` client-only tag:
+
+```
+@+example-client-tag=example-value PRIVMSG #channel :
+```
+
+---
+
+An alternate form of an empty message sent by a client with the `+example-client-tag` client-only tag, where the last parameter is omitted
+
+```
+@+example-client-tag=example-value PRIVMSG #channel
```

Comment:
As mentioned above in the diff, the only other command is NOTICE.

>even if the actual message is not in last parameter anymore, but in the tag

This isn't about moving the actual message to a tag. If it's a message that has to be displayed as such, it still belongs in the last parameter. This is about opening another channel of communication that is only visible to clients that support this spec and won't share the same behavior of public messages. See for example #289, about reactions. Another example is typing notifications.